cmd/buildah/mount.go

```diff
@@ -41,14 +48,15 @@ func init() {
 
 	flags := mountCommand.Flags()
 	flags.SetInterspersed(false)
+	flags.BoolVar(&outputJSON, "json", false, "output in JSON format")
```

Comment:
Make a global mountOptions with outputJson as a param.

type mountOptions struct {
	json        bool
}

Then no need to pass json flag as an option.

Reply:
Done! I've omitted `noTruncate` from the struct as the value of that flag is not actually used in the command implementation, but let me know if you'd prefer that it be included in the `mountOptions` struct as well.